api/nntrainer-api-common.h

```diff
@@ -27,8 +27,15 @@ extern "C" {
  * @since_tizen 6.0
  */
 typedef enum {
-  ML_TRAIN_LAYER_TYPE_INPUT = 0,    /**< Input Layer */
-  ML_TRAIN_LAYER_TYPE_FC,           /**< Fully Connected Layer */
+  ML_TRAIN_LAYER_TYPE_INPUT = 0,  /**< Input Layer */
+  ML_TRAIN_LAYER_TYPE_FC,         /**< Fully Connected Layer */
```

Comment:
you can consider making it alphabetical.

Reply:
Input and FC is released so I guess it is not possible?

Reply:
We do not any applications with nntrainer, so IMO it can be done.
Please take advice from @again4you on this.

Reply:
If we keep making it sorted, this will break someday let alone additional ACR we have to go thorugh.
As we are going to keep adding layers. Let's keep it as it is if there's no good reason.

Reply:
works.